recurly/__init__.py

```diff
@@ -418,6 +418,22 @@ class Adjustment(Resource):
     xml_attribute_attributes = ('type',)
     _classes_for_nodename = {'tax_detail': TaxDetail,}
 
+    # This can be removed when the `original_adjustment_uuid` is moved to a link
+    def __getattr__(self, name):
+        if name == 'original_adjustment':
+            try:
+                uuid = super(Adjustment, self).__getattr__('original_adjustment_uuid')
+            except (AttributeError):
+                return super(Adjustment, self).__getattr__(name)
+
+            def make_relatitator(uuid):
+                def relatitator():
+                    return Adjustment.get(uuid)
+                return relatitator
```

Comment:
What if you used a lambda here instead of a function definition? I could see arguments either way really.

```python
return lambda: Adjustment.get(uuid)
```